README.md

```diff
@@ -122,6 +122,12 @@ public class PetResource {
 
 See the [integration-tests](integration-tests) module for more information of how to use this extension. Please be advised that the extension is on experimental, early development stage.
 
+## RESTEasy Reactive and Classic support
+
+This extension supports both RESTEasy implementations (Classic and Reactive). The generated code is based on the implementation that is present in the application's dependencies.  
+
+For both implementations, the generated code is always blocking code.
+
```

Comment:
Thanks, but I guess more changes are required.

In section ``OAuth2 Authentication`` the Quarkus OIDC Client Filter Extension for ``classic`` has been mentioned only. Obviously, for ``reactive`` we need another one.

In section ``Sending multipart/form-data`` it's described that the dependency ``quarkus-reasteasy-multipart`` is required, which is not true for the ``reactive`` use case. The reactive client supports ``multipart`` out of the box. In addition, the provided examples are just valid for ``classic``, not ``reactive``.

In section ``Known Limitations`` the ``no reactive support`` needs to be removed. 

I hope I got everything

Reply:
@chberger I've copied the changes you pointed out from your PR. Now I think it's all there.

@ricardozanini the features are the same for reactive and classic. They only have slight differences, which are now pointed out.